Makefile

```diff
@@ -1,30 +1,30 @@
-# CFLAGS = -Xcc -DMONGOC_COMPILATION -Xcc -DBSON_COMPILATION -Xcc -ISources/libbson -Xcc -ISources/libbson/include -Xcc -ISources/libbson/generated -Xcc -ISources/libmongoc/generated
-# LDFLAGS = -Xlinker -lsasl2 -Xlinker -lz
-CFLAGS =
-LDFLAGS =
-
-# If FILTER is not provided, a default filter of `MongoSwiftTests` will be used.
-# Else, any test matching the filter in *either target* (MongoSwiftTests or MongoSwiftBenchmarks) will run.
+# if provided, FILTER is used as the --filter argument to `swift test`. 
 ifdef FILTER
 	FILTERARG = --filter $(FILTER)
 else
-	FILTERARG = --filter MongoSwiftTests
+	FILTERARG =
 endif
 
+define check_for_gem
+	gem list $(1) -i > /dev/null || gem install $(1) || { echo "ERROR: Failed to locate or install the ruby gem $(1); please install yourself with 'gem install $(1)' (you may need to use sudo)"; exit 1; }
+endef
+
 all:
-	swift build -v $(CFLAGS) $(LDFLAGS)
+	swift build -v
 
+# project generates the .xcodeproj, and then modifies it to add
+# spec .JSON files to the project
 project:
 	swift package generate-xcodeproj
-	@# use xcodeproj to add .json files to the project
-	@gem list xcodeproj -i > /dev/null || gem install xcodeproj || { echo "ERROR: Failed to locate or install the ruby gem xcodeproj; please install yourself with 'gem install xcodeproj' (you may need to use sudo)"; exit 1; }
+	@$(call check_for_gem, xcodeproj)
 	ruby add_json_files.rb
 
 test:
-	swift test -v $(CFLAGS) $(LDFLAGS) $(FILTERARG)
+	swift test -v $(FILTERARG)
 
-benchmark:
-	swift test -v $(CFLAGS) $(LDFLAGS) --filter MongoSwiftBenchmarks
+test-pretty:
+	@$(call check_for_gem, xcpretty)
```

Comment:
Forgot to include this in my original review, sorry for the extra email spam:

Should we add something to the `README.md` or `Development.md` that notifies users that they can get prettier testing on CLI if they install `xcpretty`, but that it is not mandatory?